### PR TITLE
Feature/login page

### PR DESCRIPTION
**Pull request by JoseI11**
# Summary

-Creation of the login page and its design
-Responsive aspect ok
...

# Detail
Login Page finished and waiting to be pulled to the development branch

# Evidence
Login desktop
Vite-React-TS-—-Mozilla-Firefox-2023-11-05-21-50-51
Vite-React-TS-—-Mozilla-Firefox-2023-11-05-21-52-10

# Reference.

Link to related documentation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint \"src/**/*.{ts,tsx,js,jsx}\" --fix",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
```
Comment (Fludeo) [inline package.json:9]: this should be the right 'lint' script
 "lint": "eslint \"src/**/*.{ts,tsx,js,jsx}\" --fix"

File: src/pages/Login.css
Comment (Fludeo) [inline src/pages/Login.css:1]: We should be only using tailwind. There is no need for CSS files.
Tailwind even has modifiers  (example [20px] ) for custom units.
Also, we should not be using absolute units.

**Review by Fludeo — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (JoseI11) [reply package.json:9]: Done

File: src/pages/Login.css
Comment (JoseI11) [reply src/pages/Login.css:1]: Done

**Review by Fludeo — APPROVED** [review]

**Pull request merged**